src/core/types/addons.js

```diff
@@ -167,3 +167,7 @@ export type AddonType = {
   isWebExtension: boolean,
   themeData?: ThemeData,
 };
+
+export type CreateInternalAddonOptions = {|
```

Comment:
This should be moved to `core/reducers/addons.js` right above the definition of `createInternalAddon`.